app/assets/javascripts/views/story_view.js

```diff
@@ -577,6 +571,22 @@ module.exports = FormView.extend({
           descriptionContainer
         );
     }
+
+    const $storyEstimate = this.$('[data-story-estimate]');
+    if ($storyEstimate) {
```

Comment:
Even if jQuery doesn't find it it'll still be truthy because jQuery always returns an object. You can do `if($storyEstimate.length)` instead 😄